### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by KAquib24** [community/community#134766] [GitHub Education]
### Select Topic Area

General

### GitHub Feature Area

Other

### Body

My student developer pack is not been sent to me yet it's been more than 3 days 
![Uploading Screenshot_20240805_224002.jpg…]()

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @KAquib24 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
